platform_umbrella/config/runtime.exs

```diff
@@ -3,3 +3,7 @@ import Config
 config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   level: :debug
+
+if config_env() == :prod do
+  config :logger, :default_handler, formatter: LoggerJSON.Formatters.Basic.new(metadata: [:mfa, :request_id])
```

Comment:
We can't call the function in config at compile time. It does work at runtime though.